db/db_test.cc

```diff
@@ -8810,7 +8825,7 @@ class RecoveryTestHelper {
     ASSERT_GT(fd, 0);
     ASSERT_EQ(offset, lseek(fd, offset, SEEK_SET));
 
-    char buf[len];
+    void* buf = alloca(len);
```

Comment:
same here, i think this leaks memory. not sure if valgrind works on windows, but it might be good to run it, if it does.